cottontaildb-cli/src/main/kotlin/org/vitrivr/cottontail/cli/entity/ImportDataCommand.kt

```diff
@@ -154,10 +151,13 @@ class ImportDataCommand(client: SimpleClient) : AbstractCottontailCommand.Entity
     ).convert { Paths.get(it) }.required()
 
     /** Flag indicating, whether the import should be executed in a single transaction or not. */
-    private val singleTransaction: Boolean by option("-t", "--transaction").flag()
+    private val singleTransaction: Boolean by option("-t", "--transaction", help="Flag indicating, whether the import should be executed in a single transaction or not.").flag()
+
+    /** Flag indicating, whether the import should be fuzzy: Ignore corrupt entries and just continue */
+    private val fuzzy: Boolean by option("--fuzzy", help="Flag indicating, whether the import should be fuzzy: Ignore corrupt entries and just continue").flag()
```

Comment:
I'd suggest renaming this to force-import or something, fuzzy means something different

Reply:
Suggestions are welcome; i do aggree that `fuzzy` is a misnomer in this context. `--best-effort` should be a little more expressive